resources/views/about-us.blade.php

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
Why don't you extend the current layout ? Please check how `dashboard.blade.php` is implemented, with the `@section('content')`.